docs/getting_started/ros_interface/ros1.rst

```diff
@@ -2,68 +2,90 @@
 ROS 1 Getting Started
 =====================
 
-Install ROS
-===========
+Installing ROS 1
+================
 
 .. note::
 
   The ROS 1 Interbotix SLATE packages are only compatible with Noetic on Ubuntu 20.04.
 
 Follow the `Ubuntu install of ROS Noetic guide on the ROS Wiki`_.
-Once installed, make sure to follow steps 1.6 and 1.6.1 to set up and install the development and dependency toolchains.
 
 .. _`Ubuntu install of ROS Noetic guide on the ROS Wiki`: https://wiki.ros.org/noetic/Installation/Ubuntu
 
-Install the Interbotix ROS Slate Packages
-=========================================
+Configuring Port Permissions
```

Comment:
While we're dealing with the serial port, you may want to add the bit from troubleshooting about uninstalling `brltty`.